content/docs/specification/content.md

```diff
@@ -320,3 +320,32 @@ The `proxy` type let's you proxy your requests to a specific upstream. It reads
 - Mandatory
 - Permitted values: "absolute URL"
 - Example: "to=https://example.com/subpage/"
+
+---
+
+# Wildcard Records
+
+A wildcard DNS record is a record in a DNS zone that will match requests for non-existent domain names. For example you can specifiy a record that matches all the subdomains that don't have a specific TXT record.
+
+Take a look at these sample records:
+
+```
+
+example.test.                    IN A   127.0.0.1
+_redirect.example.test.          IN TXT "v=txtv0;to=http://example.com;type=host;code=302"
+_redirect.specific.example.test. IN TXT "v=txtv0;to=http://specific.test;type=host;code=302"
+_redirect._.example.test.        IN TXT "v=txtv0;to=http://wildcard.test;type=host;code=302"
+
+```
+
+All the requests for a subdomain of `example.test` like `test.example.test` would get redirected to `http://wildcard.test` by the last record because they don't have a specific record. Expect for the `specific.example.test` subdomain which has a specific record for itself.
```

Comment:
```suggestion
All the requests for a given subdomain of `example.test` such as `test.example.test` would use the wildcard record of the parent zone `_redirect._.example.test`.

In the case a more specific record is available it will take be used.
For a specific record to work a specific subdomain needs to be setup as well. This is based on DNS specifics.
With `specific.example.test` and `_redirect.specific.example.test` set it will be used instead of the above wildcard.
```